packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -212,21 +212,24 @@ class MiniCart extends Component {
     };
 
     get miniCartInner() {
-        const { checkout, productList, props } = this;
+        const { checkout, productConfirm, productList, props, state } = this;
```

Comment:
I don't see where `productConfirm` is defined on `this`.

The only two references I see are here and line 223.

Reply:
@supernova-at, i agree with you, i will remove these references.